src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,97 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+
+    private Node<K, V>[] table = new Node[DEFAULT_CAPACITY];
+    private double loadFactor = 0.75;
+    private int size = 0;
 
     @Override
     public void put(K key, V value) {
-
+        Node<K, V> node = new Node<>(hash(key), key, value, null);
+        int index = Math.abs(hash(key) % table.length);
+        Node<K, V> pointer = table[index];
+        if (table[index] == null) {
+            table[index] = node;
+            size++;
+            if (size > table.length * loadFactor) {
+                resize(table);
+            }
```

Comment:
```suggestion
            if (size > table.length * loadFactor) {
                resize(table);
            }
```

Reply:
Move it to a separate method, because use it multiple times.
And in this place it would be good to check first, then everything else.